update-edpm.yml

```diff
@@ -20,27 +20,13 @@
   tags:
     - update
 
-- name: Import run tempest test playbook
+- name: Import run test playbook
```

Comment:
Will be there option to run both first tempest and tobiko after removal of tobiko role import? Or it will be one or another to choose from?

Reply:
ack, thank you